typescript-modal/typescript-modal-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# typescript-modal-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the `<dialog>` element used for?
+
+Create pop-up dialogs or modal windows on a webpage
+
+- How do you display a modal dialog?
+
+showModal()
+
+- How do you hide a modal dialog?
+
+close()
+
+- What is the difference between the `showModal` & `show` methods?
+
+showModal shows the dialog box as a modal and show displays the dialog box
```

Comment:
And with `showModal()` the user not to be able to interact with the rest of the screen behind it, and force focus within modal